modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -828,4 +832,34 @@ public static void inBusyLock(IgniteSpinBusyLock busyLock, Runnable fn) {
             busyLock.leaveBusy();
         }
     }
+
+    /**
+     * Collects all the fields of given class which are defined as a public static within the specified class.
+     *
+     * @param sourceCls The class to lookup fields in.
+     * @param targetCls Type of the fields of interest.
+     * @return A mapping name to property itself.
+     */
+    public static <T> List<T> collectStaticFields(Class<?> sourceCls, Class<? extends T> targetCls) {
```

Comment:
I'm not sure that will be used anymore except specific of ConfigurationToSchemaDescriptorConverter. Should we move it to converter class?

Reply:
There is one more place actually. PLease take a look at `org.apache.ignite.internal.sql.engine.property.PropertiesHelper#createPropsByNameMap`

Reply:
Now it so :) Thanks